src/amber/cli/templates/app/config/initializers/import_map.cr.ecr

```diff
@@ -0,0 +1,15 @@
+require "asset_pipeline"
+
+FRONT_LOADER = AssetPipeline::FrontLoader.new(js_source_path: Path["src/javascript"], js_output_path: Path["public"]) do |import_maps|
```

Comment:
wdyt of `assets/javascript` instead?

Reply:
I kicked that around. If the deployment doesn't require the entire code base be cloned, then it makes total sense.
But, wouldn't most dockerized/containerized deploys mean the entire code base was cloned into the image and then deployed?

Reply:
I would just copy the `assets` directory to the docker image instead of `src/javascript`.  I think it will actually make creating the image easier.